### ISSUE TITLE
No matching plugin found

### ISSUE THREAD

**Issue by HampterDev** [avast/retdec#1161]
Hey I'm trying to use the decompiler but it exits right after the start:
![image](https://github.com/avast/retdec/assets/131729366/12a84adf-b88a-422d-a4f1-f652052d19df)

I built retdec with MSVC

**Comment by Nitr0-G** [comment]
> Hey I'm trying to use the decompiler but it exits right after the start:
> 
> ![image](https://user-images.githubusercontent.com/131729366/245135872-36c95649-2331-4bde-a4f2-20688e4c5a22.png)
> 
> I ran retdec-unpacker.exe and verified that's where those logs come from `%errorlevel%` equals to 1 which is `EXIT_CODE_NOTHING_TO_DO, ///< There was not found matching plugin.`
> 
> I built retdec with MSVC Is there anything I'm missing to do in order to use it?
> 
> Thanks for the help
> 
> Edit:
> 
> I tried upx version 4.0.2 on the command line, but it doesn't support the exe because it wasn't packed by it
> 
> ![image](https://user-images.githubusercontent.com/131729366/245149592-11fdf2e5-e542-4792-abaa-3c1a2f76fe42.png)
> 
> The more I look at it I think the binary is not packed why is it trying to unpack it anyway?

Hello! Please check this issue if I understood you correctly
https://github.com/avast/retdec/issues/1151

**Comment by HampterDev** [comment]
Does that mean I have to supply a configuration file? or rather 2

**Comment by HampterDev** [comment]
oh you are right thank you @Nitr0-G 

![image](https://github.com/avast/retdec/assets/131729366/406fa038-7c89-4c2c-b897-f136eacaee5f)

I placed this file and now it goes further

**Comment by Nitr0-G** [comment]
> oh you are right thank you @Nitr0-G
> 
> ![image](https://user-images.githubusercontent.com/131729366/245447568-406fa038-7c89-4c2c-b897-f136eacaee5f.png)
> 
> I placed this file and now it goes further

Yes, but I still advise you to put the entire "share" folder next to it
